### ISSUE TITLE
Add API to list orphaned packages

### ISSUE THREAD

**Issue by gaborcsardi** [metacran/crandb#41]

**Comment by maelle** [comment]
is something that needs to be added to the web API or the package?

**Comment by gaborcsardi** [comment]
both

**Comment by maelle** [comment]
Ok. if I were to try and add that to the API, where would I look? https://github.com/metacran/crandb/blob/master/inst/app.js?

**Comment by gaborcsardi** [comment]
Yeah, and you would need to run a couchdb server, to write the query.

**Comment by maelle** [comment]
Ok, if I try and succeed I can add some contributing guidelines then :grin:
